src/components/TxDetailsCard.js

```diff
@@ -0,0 +1,106 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+// @flow
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, Platform, StyleSheet, TouchableNativeFeedback, TouchableOpacity } from 'react-native';
+import colors from '../colors';
+import Card from './Card';
+import AccountIcon from './AccountIcon';
+
+const WEI_IN_ETH = 1000000000000000000;
```

Comment:
Can use this: https://github.com/parity-js/api/blob/master/src/util/wei.js#L31-L33

Reply:
yeah, I was about to do it, thanks